pkg/apis/shipper/v1alpha1/types.go

```diff
@@ -206,6 +206,7 @@ type AchievedStep struct {
 type ReleaseConditionType string
 
 const (
+	ReleaseConditionTypeClustersChosen   ReleaseConditionType = "ClustersChosen"
```

Comment:
I think we never really distinguished 2 states of a release object: clusters chosen and scheduled. Does it make sense to keep "scheduled" around?

Reply:
"Scheduled" still means that the target objects were created. We definitely need a new name, but I don't think that the actual condition would go away, unless we move it to be inside the actual strategy, but that's way too much work for the context of this PR. I'd happily rename it though, I just haven't figured out a better name yet.

Reply:
I've finished reading and your words totally make sense to me. Naming is what confused me. Thanks.